app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,14 @@
+class PagesController < ApplicationController
+  def index
+  end
+
+  def about
+  end
+
+  def meetup
+  end
+
+  def contact
+
```

Comment:
We can get rid of the space here inside the contact definition (has no impact on the code, just looks cleaner)